app/api.js

```diff
@@ -60,14 +60,17 @@ const USER_FIELDS = `
 `
 
 export function fetchUser() {
-  // TODO extract constants
   const query = `{
-    user { ${USER_FIELDS} }
+    user {
+      profile {
+        ${USER_PROFILE_FIELDS}
+      }
+    }
   }
   `
   const url = `${BASE_URL}${GRAPHQL}?query=${query}`
   return ftch(url, { ...credentials() })
-    .then(res => Promise.resolve(res.data.user))
+    .then(res => Promise.resolve(res.data.user.profile))
```

Comment:
I don't think you need Promise.resolve in these cases. As far as I know this is done implicitly.